packages/editor-ui/src/components/WorkflowActivator.vue

```diff
@@ -100,6 +111,20 @@ async function displayActivationError() {
 		duration: 0,
 	});
 }
+
+watch(
+	() => props.workflowActive,
+	(workflowActive) => {
```

Comment:
There's an edge case to be aware of here. Sometimes in n8n, this will turn true/active for a second, then turn back false/inactive. This happens if n8n is unable to activate trigger node.
So the first POST request to activate works, but a websocket event later comes to disable the active workflow. 

In this case, the user will see this notification, but the workflow will be inactive.
It's probably not a big issue, but good to be aware of. 

Also would be good to check the workflows page to make sure there's no issues. We probably don't need to show the notification there, but we have the same component there.